tests/tests-core/src/main/java/com/gentics/mesh/search/IndexSearchModeTest.java

```diff
@@ -0,0 +1,36 @@
+package com.gentics.mesh.search;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.stream.Collectors;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+import com.gentics.mesh.etc.config.search.IndexSearchMode;
+import com.gentics.mesh.test.ElasticsearchTestMode;
+import com.gentics.mesh.test.MeshTestSetting;
+import com.gentics.mesh.test.TestSize;
+
+@RunWith(Parameterized.class)
+@MeshTestSetting(startServer = true, testSize = TestSize.FULL)
+public class IndexSearchModeTest extends NodeSearchEndpointCTest {
```

Comment:
This particular test uses a data set good enough for the index search pagination test.